sources/packages/api/src/esdc-integration/msfaa-integration/msfaa-files/msfaa-file-detail.ts

```diff
@@ -49,7 +49,7 @@ export class MSFAAFileDetail implements MSFAARequestFileLine {
     record.appendWithEndFiller(this.addressLine1, 40, SPACE_FILLER);
     record.appendWithEndFiller(this.addressLine2, 40, SPACE_FILLER);
     record.appendWithEndFiller(this.city, 25, SPACE_FILLER);
-    record.appendWithEndFiller(this.province, 4, SPACE_FILLER);
+    record.appendWithEndFiller(this.provinceState, 4, SPACE_FILLER);
```

Comment:
If `provinceState` is not going to be populated by countries other than Canada we should add a null check.
```typescript
record.appendWithEndFiller(this.provinceState ?? "", 4, SPACE_FILLER)
```